docs/cloud/dbaas/backups.md

```diff
@@ -11,23 +11,23 @@ after the last backup. All backups are stored for 90 days after which they get a
 deleted (not yet implemented). In the future, it will only be possible to delete backups automatically,
 users will not be able to manually delete their own backups.
 
-The backups are stored encrypted in Allas. 
+The backups are stored encrypted in Allas.
 
 You can manage your backups from the CLI and the web-interface.
 
 ## Risks
 
 Be aware that the backup feature is still in development and should not be used for anything remotely sensitive or important.
 
-* Backups are stored encrypted in Allas. This is worth to keep in mind if you want to take your own backup (you should not store it in Allas).
+* Backups are stored encrypted in Allas. This is worth to keep in mind if you want to take your own backup (you should probably store them somewhere else than Allas nor cPouta).
```

Comment:
I think this is an incorrect use of nor. My recommendation:

```suggestion
Backups are stored encrypted in Allas. This is worth keeping in mind if you want to make your own backups (you should probably store them somewhere else than Allas or cPouta).`

Reply:
Update as suggested.